docs/source/workloadConfig.rst

```diff
@@ -80,6 +80,17 @@ explicitly build this workload. This option may include arguments for the script
 However, any affects that host-init has on the resulting rootfs *will* be
 reflected in the child.
 
+bin-post
+--------------
+A script to run natively on your host (i.e., them machine where you
+invoked FireMarshal) from the workload source directory after you build the
+binary(s) for this workload but before the image(s) is/are built. This option may
+include arguments for the script, e.g. ``"bin-post" : "foo.sh bar baz"``.
+
```

Comment:
it might be nice to include some examples of why you would do this. It took some thinking on our part to figure out that we needed this for building drivers. Perhaps later we can include a more detailed example of including custom kernel modules (not needed for this PR). Some reasons for post-bin:
* Building kernel modules against the specific kernel and kconfig that are being used.
* Analyzing the kernel binary to extract debugging or tracing information

It might be good to also document what the user can expect at this phase:
* The binary will be built, but the image will not
* The kernel configuration used to build the kernel will be at FIREMARSHAL_LINUX_SRC/.config
* Any build artifacts generated by building the kernel will continue to be present (we will not run 'make clean' or equivalent in the kernel source).